Gruntfile.js

```diff
@@ -179,12 +157,10 @@ module.exports = function(grunt) {
   // Testing tasks
   grunt.registerTask('test', ['shell:unit', 'shell:accept']);
   grunt.registerTask('unit', ['shell:unit']);
-  grunt.registerTask('accept', ['env:accept', 'shell:accept']);
 
   // Coverate tasks
-  grunt.registerTask('coverage', ['shell:coverage_unit', 'shell:coverage_accept']);
+  grunt.registerTask('coverage', ['shell:coverage_unit']);
   grunt.registerTask('coverage-unit', ['shell:coverage_unit']);
```

Comment:
coverage and coverage-unit do the same now, so you can probably delete coverage-unit